src/handwritten/grpc_session_options.py

```diff
@@ -0,0 +1,67 @@
+from enum import IntEnum
+
+
+# This constant specifies the gRPC package and service used by this API.
+# Customers can pass this value to the MeasurementLink discovery service to resolve the server instance that provides this interface.
+GRPC_SERVICE_INTERFACE_NAME = 'nidaqmx_grpc.NiDAQmx'
+
+# This constant specifies the API license key required by the NI gRPC Device Server that comes with
+# MeasurementLink 2023 Q1.
+MEASUREMENTLINK_23Q1_PYTHON_API_KEY = 'DE10751B-3EE0-44EC-A93B-800E6A3C89E4'
```

Comment:
We have reused the same key from nimi-python, @bkeryan , Please confirm if this is okay

Reply:
I discussed this with @zhindes and we are going to use a different GUID. I used GuidGen to generate one: `'90EDC55C-DE8B-4894-A28C-223BD7171383'`

![image](https://user-images.githubusercontent.com/17914399/233191191-18523450-0f6a-42ef-82df-3c4e3cc30df0.png)

Reply:
Thank you Brad. Used the given api constant and key.